R/plumb-block.R

```diff
@@ -250,7 +250,8 @@ plumbBlock <- function(lineNum, file, envir = parent.frame()){
     parsers = rev(parsers),
     assets = assets,
     params = rev(params),
-    comments = paste0(rev(comments), collapse = " "),
+    comments = paste0(rev(comments)[1], collapse = " "),
+    description = paste0(rev(comments)[-1], collapse = " "),
```

Comment:
Should we collapse using a new line? Feels odd to have two multi-line paragraphs result into a large, single paragraph.

```suggestion
    description = paste0(rev(comments)[-1], collapse = "\n"),
```

Reply:
Makes sense